ui/lib/sync/addon/components/secrets/page/destinations/destination/details.hbs

```diff
@@ -3,7 +3,24 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<Secrets::DestinationsHeaderTabs @destination={{@destination}} />
+<SyncHeader
+  @icon={{@destination.icon}}
+  @title={{@destination.name}}
+  @breadcrumbs={{array
```

Comment:
love me some breadcrumbs that don't have to account for directories and their separate routes :)

Reply:
right?!